B/BitConverter/Package.toml

```diff
@@ -1,3 +1,3 @@
 name = "BitConverter"
 uuid = "3a3ce9e8-98e7-11e9-0fa0-055639f146d3"
-repo = "https://github.com/roshii/BitConverter.jl.git"
+repo = "https://gitlab.com/braneproject/bitconverter.jl.git"
```

Comment:
Shouldn't the package name still be `BitConverter`?

Reply:
https://github.com/JuliaRegistries/General/pull/1817/commits/6b37209c941569278e19e2f689e8f0178ca3ab27